tensorflow/python/ops/math_ops.py

```diff
@@ -1685,6 +1685,11 @@ def matmul(a,
                            [229 244]],
                           [[508 532]
                            [697 730]]]
+
+  # Since python >= 3.5 and TensorFlow >= 1.1, the @ operator is supported.
```

Comment:
Don't mention TensorFlow 1.1.  This comment is unlikely to be read by someone reading the source code for an earlier version of TensorFlow.